web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
Well. Lot of noise ! Please @bealdav, just split this file (or let me do it !) ;-)
1/ It's not the place to discuss to change this rule
2/ having a ir_ui_view class in a file named model.py is confusing. Not really "productive" as you say

LGTM otherwise. thanks.
Regards.